src/Uno.Toolkit.UI/Controls/ZoomContentControl/ZoomContentControl.Properties.cs

```diff
@@ -0,0 +1,477 @@
+
+	/// Gets or sets the maximum zoom level allowed.
+	public double MaxZoomLevel
+	{
+		get => (double)GetValue(MaxZoomLevelProperty);
+		set => SetValue(MaxZoomLevelProperty, value);
+	}
+
+	#endregion
+
+	#region DependencyProperty: PanWheelRatio
+
+	/// Identifies the PanWheelRatio dependency property.
+	public static DependencyProperty PanWheelRatioProperty { get; } = DependencyProperty.Register(
+		nameof(PanWheelRatio),
+		typeof(double),
+		typeof(ZoomContentControl),
+		new PropertyMetadata(0.25d));
+
+	/// Gets or sets the ratio used for panning with the mouse wheel.
+	public double PanWheelRatio
+	{
+		get => (double)GetValue(PanWheelRatioProperty);
+		set => SetValue(PanWheelRatioProperty, value);
+	}
+
+	#endregion
+
+	#region DependencyProperty: ScaleWheelRatio
+
+	/// Identifies the ScaleWheelRatio dependency property.
+	public static DependencyProperty ScaleWheelRatioProperty { get; } = DependencyProperty.Register(
+		nameof(ScaleWheelRatio),
+		typeof(double),
+		typeof(ZoomContentControl),
+		new PropertyMetadata(0.0006d));
+
+	/// Gets or sets the ratio used for scaling the zoom level with the mouse wheel.
+	public double ScaleWheelRatio
+	{
+		get => (double)GetValue(ScaleWheelRatioProperty);
+		set => SetValue(ScaleWheelRatioProperty, value);
+	}
+
+	#endregion
+
+	#region DependencyProperty: IsActive
+
+	/// Identifies the IsActive dependency property.
+	public static DependencyProperty IsActiveProperty { get; } = DependencyProperty.Register(
+		nameof(IsActive),
+		typeof(bool),
+		typeof(ZoomContentControl),
+		new PropertyMetadata(true, OnIsActiveChanged));
+
+	/// Gets or sets a value indicating whether the control is active.
+	public bool IsActive
+	{
+		get => (bool)GetValue(IsActiveProperty);
+		set => SetValue(IsActiveProperty, value);
+	}
+
+	#endregion
+
+	#region DependencyProperty: IsZoomAllowed
+
+	/// Identifies the IsZoomAllowed dependency property.
+	public static DependencyProperty IsZoomAllowedProperty { get; } = DependencyProperty.Register(
+		nameof(IsZoomAllowed),
+		typeof(bool),
+		typeof(ZoomContentControl),
+		new PropertyMetadata(true));
+
+	/// Gets or sets a value indicating whether zooming is allowed.
+	public bool IsZoomAllowed
+	{
+		get => (bool)GetValue(IsZoomAllowedProperty);
+		set => SetValue(IsZoomAllowedProperty, value);
+	}
+
+	#endregion
+
+	#region DependencyProperty: IsHorizontalScrollBarVisible
+
+	/// Identifies the IsHorizontalScrollBarVisible dependency property.
+	public static DependencyProperty IsHorizontalScrollBarVisibleProperty { get; } = DependencyProperty.Register(
+		nameof(IsHorizontalScrollBarVisible),
+		typeof(bool),
+		typeof(ZoomContentControl),
+		new PropertyMetadata(true));
+
+	/// Gets or sets a value indicating whether the horizontal scrollbar is visible.
+	public bool IsHorizontalScrollBarVisible
+	{
+		get => (bool)GetValue(IsHorizontalScrollBarVisibleProperty);
+		set => SetValue(IsHorizontalScrollBarVisibleProperty, value);
+	}
```

Comment:
We are very frequently setting this DP ourselves. It feels like a setter shouldn't be public.